types/mjml-react/index.d.ts

```diff
@@ -125,10 +130,26 @@ export class MjmlStyle extends React.Component<{ children?: string | undefined,
 export class MjmlTitle extends React.Component<{ children?: string | undefined }> { }
 
 // mj-accordion
-export class MjmlAccordion extends React.Component<MjmlAccordionElementProps> { }
+export interface MjmlAccordionProps {
+    children?: React.ReactNode;
+    border?: string | undefined;
+    containerBackgroundColor?: React.CSSProperties['backgroundColor'] | undefined;
+    fontFamily?: string | undefined;
+    iconAlign?: string | undefined;
+    iconHeight?: string | undefined;
+    iconPosition?: 'left' | 'right' | undefined;
+    iconUnwrappedAlt?: string | undefined;
+    iconUnwrappedUrl?: string | undefined;
+    iconWidth?: string | undefined;
+    iconWrappedAlt?: string | undefined;
+    iconWrappedUrl?: string | undefined;
+}
```

Comment:
It looks like there's a missing prop `backgroundColor` from the `MjmlAccordionElementProps`.

I checked the docs and it's not in the Accordion Props, however, making this change will cause anyone upgrading to deal with this type error if it's used anywhere by accident. This might lead someone to keep a stale version of the types. I personally would prefer people are able to upgrade but give them a deprecation message. I don't know what the DefinitelyTyped mentality is on this so I'm fine with this either way.


```suggestion

    /**
     * @deprecated backgroundColor doesn't exist on the mj-accordion element.
     * It is included for backwards compatibility with an erroneous version of the mjml-react types.
     */
    backgroundColor?: React.CSSProperties['backgroundColor'] | undefined;
}
```